assets/javascripts/discourse/lib/ai-streamer.js

```diff
@@ -113,13 +113,16 @@ class PostUpdater extends StreamUpdater {
     await this.setCooked(cookedElement.innerHTML);
   }
 
-  async setCooked(value) {
-    this.post.set("cooked", value);
+  async setCooked(cooked) {
+    this.post.set("cooked", cooked);
 
     const oldElement = this.postElement.querySelector(".cooked");
 
-    await loadScript("/javascripts/diffhtml.min.js");
-    window.diff.innerHTML(oldElement, value);
+    // TODO: use `morphInner` once version 0.0.16 of morphlex is out
```

Comment:
`morphlex` doesn't support morphing between a node and a string in version 0.0.15. It has been added though - https://github.com/phlex-ruby/morphlex/commit/bf208fb20535add663aaa1c7729c5f00836ae8a3